src/lib/styles/globals/controls.css

```
@@ -17,45 +18,50 @@
     text-shadow var(--motion-duration-standard) var(--motion-ease-standard),
     opacity var(--motion-duration-standard) var(--motion-ease-standard);
   background: transparent;
+  box-shadow: inset 0 0 0 1px var(--lattice-btn-ring);
 }
 
 .lattice-btn--primary {
   --lattice-btn-tone: var(--accent-cyan);
   --lattice-btn-tone-rgb: var(--rgb-running);
-  border: 1px solid var(--accent-cyan);
+  --lattice-btn-ring: var(--accent-cyan);
   color: var(--accent-cyan);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)